examples/resnet-tracing.cpp

```diff
@@ -0,0 +1,157 @@
+/**
```

Comment:
Can you just fold a tracing option into image-classifier instead of adding another resnet sample?